data/coalition/coalition friend of arthropods.txt

```diff
@@ -0,0 +1,905 @@
+			`	Rounded Purple B14 performs a strange beetle bow to you before departing to go about their business. No rest for the project lead you suppose. The aide makes sure you safely find your way to the exit of the massive structure and wishes you the best, as well as transferring you the promised payment from delivering the bacterial soup.`
+				decline
+
+
+
+mission "Grubbing Crisis Resolution A"
+	invisible
+	landing
+	to offer
+		has "grub crisis timer"
+		not "A Grateful Head: Offered"
+	on offer
+		event "grubbing ground resolved"
+
+
+event "grubbing ground resolved"
+
+
+
+mission "Grubbing Crisis Resolution B"
+	invisible
+	landing
+	to offer
+		has "grub crisis timer"
+		has "A Grateful Head: Offered"
+		not "clean grubbing resolution"
+	on offer
+		event "grubbing time running out"
+		event "grubbing ground resolution delayed" 30
+		conversation
+			`	You receive an urgent message from Rounded Purple B14; "Delaying weather cycle we are. Opportunity window closing though. Hurry please."`
+
+
+event "grubbing time running out"
+
+event "grubbing ground resolution delayed"
+
+
+
+mission "Grubbing Crisis Resolution C"
+	invisible
+	landing
+	to offer
+		has "Witness the Birthing: Offered"
+		not "Double Haze Soup: Failed"
+	on offer
+		event "clean grubbing resolution"
+
+
+event "clean grubbing resolution"
+
+
+
+phrase "grub savior invertebrates"
+	word
+		`You deliver the invertebrates to a harried Kimek scientist on <planet>'s spaceport. A host of assistants inspect the shipment and implore you to bring more before transferring <payment> and rushing off.`
+
+phrase "grub savior caged small animals"
+	word
+		`You deliver the small animals to a harried Kimek scientist on <planet>'s spaceport. A host of assistants inspect the shipment and implore you to bring more before transferring <payment> and rushing off.`
+
+phrase "grub savior seed crates"
+	word
+		`You deliver the seed crates to a harried Kimek scientist on <planet>'s spaceport. A host of assistants inspect the shipment and implore you to bring more before transferring <payment> and rushing off.`
+
+phrase "grub savior caged birds"
+	word
+		`You deliver the birds to a harried Kimek scientist on <planet>'s spaceport. A host of assistants inspect the shipment and implore you to bring more before transferring <payment> and rushing off.`
+
+phrase "grub savior food stock biodegradables"
+	word
+		`You deliver the food stock biodegradables to a harried Kimek scientist on <planet>'s spaceport. A host of assistants inspect the shipment and implore you to bring more before transferring <payment> and rushing off.`
+
+
+
+#Crisis Jobs for the Job Board.
+mission "FoA: Transport Invertebrates 50t"
+	job
+	repeat
+	name "Transport 50t of Invertebrates to <planet>"
+	description "Transport this <cargo> to the world of <planet>, where it will be used to address the crisis with their new grubbing ground."
```

Comment:
```suggestion
	description "Transport <cargo> to the world of <planet>, where it will be used to address the crisis with their new grubbing ground."
```
I think this is what standard job descriptions say. Not sure though, so I'm only commenting on this job.